challenge_7/python/alexbotello/missing_num.py

```diff
@@ -0,0 +1,18 @@
+def find_missing_num(array):
+	
+	sum_of_array = 0
+	sum_of_indexes = 0
+
+	for a in array:
+		sum_of_array += a
+	for b in range(1, len(array) + 1):
```

Comment:
This works, but for the future, it may be useful to know that `1 + 2 + ... + N = N(N+1)/2`